src/bin/pg_autoctl/pghba.c

```diff
@@ -53,7 +54,15 @@ pghba_ensure_host_rule_exists(const char *hbaFilePath,
 	char *includeLine = NULL;
 	PQExpBuffer newHbaContents = NULL;
 
-	hbaLineEnd += sprintf(hbaLineEnd, "host ");
+	if (ssl)
+	{
+		hbaLineEnd += sprintf(hbaLineEnd, "hostssl ");
```

Comment:
whole string usage is being changed in another PR.

Reply:
Yeah but we don't know yet which PR is making it first, and it might be the SSL one anyway.